src/platform/src/Bridge/ElevenLabs/ElevenLabsClient.php

```diff
@@ -86,16 +85,25 @@ private function doTextToSpeechRequest(Model $model, array|string $payload, arra
         }
 
         $voice = $options['voice'] ??= $model->getOptions()['voice'];
+        $stream = $options['stream'] ??= $model->getOptions()['stream'] ?? false;
+
+        $url = $stream
+            ? \sprintf('%s/text-to-speech/%s/stream', $this->hostUrl, $voice)
+            : \sprintf('%s/text-to-speech/%s', $this->hostUrl, $voice);
 
-        return new RawHttpResult($this->httpClient->request('POST', \sprintf('%s/text-to-speech/%s', $this->hostUrl, $voice), [
+        $response = $this->httpClient->request('POST', $url, [
             'headers' => [
                 'xi-api-key' => $this->apiKey,
             ],
             'json' => [
                 'text' => $payload['text'],
                 'model_id' => $model->getName(),
             ],
-        ]));
+        ]);
+
+        return $stream
+            ? new RawHttpResult($this->httpClient->stream($response))
```

Comment:
in my understanding this call of `stream` method break the concurrency feature and should be shifted to the `ElevenLabsResultConverter`